### PR TITLE
Add debug mode

### PR DESCRIPTION
**Pull request by amkisko**
Implements basic debug logging of child process output

### CODE REVIEW

**Comment by n-rodriguez** [comment]
Hi there! Any news?

**Comment by rhys-vdw** [comment]
Hi @n-rodriguez. I no longer maintain this project as I'm not at UsabilityHub anymore.

FYI @PikachuEXE ☝️

File: index.js [hunk 1]
```diff
@@ -91,6 +91,7 @@ function transformSource (runner, config, source, map, callback) {
 
   var dataBuffers = []
   child.stdout.on('data', function (data) {
+    if (config.debug) { console.log(data) }
```
Comment (PikachuEXE) [inline index.js:94]: Is there a `console.debug` method? Maybe it should be used instead?

**Comment by PikachuEXE** [comment]
Oh it's up to me now?
Who is gonna handle the releases?

I am happy to help as much as I can
But with my limited knowledge/understanding I can only handle smaller changes :)

File: index.js (hunk 1 above)
Comment (rhys-vdw) [reply index.js:94]: There is only `console.log` and `console.error` for STDOUT and STDERR respectively.

**Comment by rhys-vdw** [comment]
@PikachuEXE obviously you don't have to take on the responsibility of maintaining it. I was just giving you a heads up as you're the most active contributor that I remember. I can follow up release permissions if you'd like them. I personally have not worked with Rails for over 6 months so I'm not in a great position to maintain either.

**Comment by PikachuEXE** [comment]
I am happy to review small changes and release new versions
The last "big" update might be #78

**Pull request merged**